sdk.mjs

```diff
@@ -201,11 +207,11 @@ export const getThirdPartyExtensions = async (
       module = module.split('<img src="')[0];
       return {
         module: {
-          name: /\[(.+)\]/.exec(module)[1],
+          name: /\[(.+)]/.exec(module)[1],
           url: /\((.+)\)/.exec(module)[1],
         },
         author: {
-          name: /\[(.+)\]/.exec(author)[1],
+          name: /\[(.+)]/.exec(author)[1],
```

Comment:
Is this a bug in xo? Note that the first `[` is escaped.

Reply:
![CleanShot 2024-03-19 at 19 39 09@2x](https://github.com/simple-icons/simple-icons/assets/8186898/7d17daaf-bcd5-4b5c-8564-519dcc7e9940)

See.

Reply:
Yes, seems to be that if there is a `\[` at the start of a `[]` group, the entire group is treated as literal, but IMHO anyways looks like unreadable and counterintuitive. Don't worry, I'll replace this regular expressions on #10637